Source/BottomView/ImageStack.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+
+public class ImageStack {
+
+  public struct Notifications {
+    public static let imageDidPushNotification = "imageDidPush:"
```

Comment:
I guess `Notification` suffix could be dropped since it's inside of `Notifications` struct.